Gemfile

```diff
@@ -42,6 +43,7 @@ group :test do
   gem "capybara", "~> 3.34"
   gem "capybara-screenshot"
   gem "cuprite"
+  gem "selenium-webdriver"
```

Comment:
The `axe-core-rspec` gem does not have any plans to support cuprite. It defaults to the selenium driver, so it is added here.

Reply:
make this a comment on line 46 please (e.g.,  # needed for axe-core-rspec)

I don't know what cuprite is for, and would welcome a comment there, fwiw.

Reply:
Done.